app/src/main/java/com/fstyle/androidtrainning/screen/detailsmovie/DetailsMovieActivity.java

```diff
@@ -32,4 +86,29 @@ protected void onStop() {
         mPresenter.onStop();
         super.onStop();
     }
+
+    @Override
+    public Integer getMovieId() {
+        return getIntent().getIntExtra(Constant.EXTRA_MOVIE_ID, 0);
+    }
+
+    @Override
+    public void onDetailsMovieSuccess(Movie movie, List<String> listGenre) {
+        String urlBackdrop = StringUtils.convertPosterPathToUrlPoster(movie.getBackdropPath());
+        String urlPoster = StringUtils.convertPosterPathToUrlPoster(movie.getPosterPath());
+        String genresCommaSeparated = StringUtils.convertListToStringCommaSeparated(listGenre);
+        Date date =
+                DateTimeUtils.convertStringToDate(movie.getReleaseDate(), DATE_FORMATT_DD_MM_YYYY);
+        String releaseDate =
+                DateTimeUtils.getStrDateTimeFormatted(date, DATE_FORMATT_DD_MMMMM_YYYY);
+
+        Glide.with(this).load(urlBackdrop).into(mImageBackdrop);
+        Glide.with(this).load(urlPoster).into(mImagePoster);
+        mTextReleaseDate.setText(releaseDate);
+        mTextGenreMovie.setText(genresCommaSeparated);
+        mTextRateMovie.setText(String.valueOf(movie.getVoteAverage()) + MAX_POINT);
```

Comment:
WARNING: [UselessStringValueOf] No need to call String.valueOf to append to a string.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#UselessStringValueOf